Remove redundant declarations to suppress deprecation warnings
See upstream issue: https://github.com/abseil/abseil-cpp/issues/1191

--- absl/container/fixed_array.h
+++ absl/container/fixed_array.h
@@ -489,12 +489,14 @@ class FixedArray {
   Storage storage_;
 };
 
+#if  __cplusplus < 201703L
 template <typename T, size_t N, typename A>
 constexpr size_t FixedArray<T, N, A>::kInlineBytesDefault;
 
 template <typename T, size_t N, typename A>
 constexpr typename FixedArray<T, N, A>::size_type
     FixedArray<T, N, A>::inline_elements;
+#endif
 
 template <typename T, size_t N, typename A>
 void FixedArray<T, N, A>::NonEmptyInlinedStorage::AnnotateConstruct(
--- absl/strings/internal/string_constant.h
+++ absl/strings/internal/string_constant.h
@@ -50,8 +50,10 @@ struct StringConstant {
                 "The input string_view must point to constant data.");
 };
 
+#if  __cplusplus < 201703L
 template <typename T>
 constexpr absl::string_view StringConstant<T>::value;  // NOLINT
+#endif
 
 // Factory function for `StringConstant` instances.
 // It supports callables that have a constexpr default constructor and a
